templates/jobs/job_detail.html

```diff
@@ -8,7 +8,7 @@
 {% block content_attributes %}with-right-sidebar{% endblock %}
 
 {% block og_title %}Job: {{ object.job_title }} at {{ object.company_name }}{% endblock %}
-{% block og-descript %}{{ object.description|escape|truncatechars:200 }}{% endblock %}
+{% block og-descript %}{{ object.description|striptags|truncatechars:200 }}{% endblock %}
```

Comment:
^ This didn't work for me locally.

I created a job with HTML description containing:
```html
hi <a href="https://example.com>example</a> there
```

This PR gave me:

<img width="405" height="218" alt="image" src="https://github.com/user-attachments/assets/42bd1295-d72f-4da4-82e1-116813aea6d3" />


```
<meta property="og:description" content="hi <a href="https://example.com>example there">
```


What did work, was first getting the `.rendered` output, and then doing `striptags` on that:

```suggestion
{% block og-descript %}{{ object.description.rendered|striptags|truncatechars:200 }}{% endblock %}
```

<img width="390" height="207" alt="image" src="https://github.com/user-attachments/assets/5a0b2a0a-7d08-4e8c-ad7a-55d4efa10abb" />


```html
<meta property="og:description" content="hi &lt;a href=&quot;https://example.com&gt;example there">
```